CONTRIBUTING.md

```diff
@@ -1,24 +1,56 @@
-# Contributing
+## Contributing
 
-We welcome any and all contributions. To get started,
+We're excited to have you contribute. Here's a step-by-step guide to help you get started.
 
-1. Make sure you have `yarn` installed. We are still using version 1.x. The installation instructions can be found here: https://classic.yarnpkg.com/lang/en/docs/install
-2. [Fork](https://github.com/alchemyplatform/aa-sdk/fork) this repo
-3. Run `yarn` to install dependencies
-4. Make changes to packages
+### Getting Started
 
-## Pull Request Checklist
+1. **Fork and Clone**: First, [fork aa-sdk repository](https://github.com/alchemyplatform/aa-sdk/fork). Then, clone your forked repo to your local machine.
```

Comment:
not sure if hyperlinks take code snippet styling. I would double check this locally.
```suggestion
1. **Fork and Clone**: First, [fork the `aa-sdk` repository](https://github.com/alchemyplatform/aa-sdk/fork). Then, clone your forked repo to your local machine.
```

Reply:
they do take the styling @avasisht23 I can confirm this!